scripts/exregional_run_prepstart.sh

```diff
@@ -835,6 +846,21 @@ else
 
 fi 
 
+#
+#-----------------------------------------------------------------------
+#
+# conduct surface surgery to get new vtype and stype
+# 
+#-----------------------------------------------------------------------
+if [ ${YYYYMMDDHH} -eq 2023030403 ] ; then
+  if [ ${cycle_type} == "spinup" ]; then
+    cp /mnt/lfs4/BMC/nrtrr/FIX_RRFS/lam/RRFS_CONUS_3km_C3359_svl/C3359_stypdom_double.nc .
+    cp /mnt/lfs4/BMC/nrtrr/FIX_RRFS/lam/RRFS_CONUS_3km_C3359_svl/C3359_vtypdom_double.nc .
+    cp sfc_data.tile7.halo0.nc sfc_data.tile7.halo0.nc_old
+    ncks -A -v stype C3359_stypdom_double.nc sfc_data.tile7.halo0.nc
+    ncks -A -v vtype C3359_vtypdom_double.nc sfc_data.tile7.halo0.nc
+  fi
+fi
```

Comment:
What's this? Line 849-863

Reply:
Will a retro need to do this? Will rrfs_a need to do this? 
Do we only need C3359_stypdom_double.nc and C3359_vtypdom_double.nc from the RRFS_CONUS_3km_C3359_svl directory? Can we use the FIX_LAM variable instead of the absolute path?

Reply:
RRFS_A may need this or may not depends on how we do the surface syncing.
Retro needs this, but need more work to make it more friendly.

We will update this piece when the model code is ready to be used for everyone.